lib/index.js

```diff
@@ -48,7 +48,13 @@ module.exports = {
 
     let self = this;
     // maintaining `baseURL` for backwards compatibility. See: http://emberjs.com/blog/2016/04/28/baseURL.html
-    let baseURL = options.liveReloadBaseUrl || options.rootURL || options.baseURL;
+    let baseURL = options.liveReloadBaseUrl ?? options.rootURL ?? options.baseURL;
+
+    // express requires absolute paths
+    if (baseURL === '') {
+      baseURL = '/';
+    }
```

Comment:
As an alternative pattern we could fall back to `'/'` not only if `baseURL` is an empty string but always when it is falsy. This would match previous behavior of Ember CLI more closely as `options.baseURL` had such a fallback.

```suggestion
    let baseURL = options.liveReloadBaseUrl || options.rootURL || options.baseURL || '/';
```

I'm not much opinionated on either or option.